video/out/vo.c

```diff
@@ -1196,9 +1196,16 @@ void vo_seek_reset(struct vo *vo)
 bool vo_still_displaying(struct vo *vo)
 {
     struct vo_internal *in = vo->in;
```

Comment:
IMO this commit should keep the `Revert "..."` message and only mention mf:// in the description